compat/test/browser/portals.test.js

```diff
@@ -178,8 +178,145 @@ describe('Portal', () => {
 		}
 
 		render(<App />, root);
-		expect(dialog.childNodes.length).to.equal(1);
+		expect(dialog.childNodes.length).to.equal(2);
```

Comment:
This is because of the empty textNode, one would expect `replaceNode` to delete it but for some reason it isn't deleted.